testrunner-toolkit/cypress/.sauce/config-ex4.yml

```diff
@@ -0,0 +1,41 @@
+apiVersion: v1alpha
+kind: cypress
+sauce:
+  region: eu-central-1 # or us-west-1
+  concurrency: 3
+  metadata:
+    name: Cypress Cross Browser tests
+    tags:
+      - cypress
+      - cross-browser
+    build: "Cypress: Cross Browser - $DATE"
+docker:
+  fileTransfer: mount
+cypress:
+  version: 7.3.0
+  configFile: "cypress.json"  # We determine related files based on the location of the config file.
+suites:
+  # Chrome v91
+  - name: "Swag Labs Login Chrome 91"
+    browser: "chrome"
+    browserVersion: "91.0"
```

Comment:
Can we put dynamic versions in here by using `latest` instead of fixed versions

Reply:
Done

Reply:
I run on latest, latest-1, latest-2. Really nice :-)